### PR TITLE
Adding token, zone, and check usage to README

### PR DESCRIPTION
**Pull request by itzg**
* also splits out the developer specific parts of the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,136 +96,87 @@ or specify a custom confiuration file location:
...
 
-First, be your own certificate authority by filling out the prompts when running:
+## Locate or create a monitoring token
 
-```
-docker run -it -v $(pwd)/data/ca:/ca itzg/cert-helper \
-  init
-```
+In order for your poller instance to authenticate against your Rackspace Monitoring account you will either need to
+obtain an existing token from your account or a create a new one. _The later is a good practice in order to delineate
```
Comment (jjbuchan) [inline README.md]: `later` vs. `latter` and `or a create`

File: README.md [hunk 2]
```diff
@@ -88,136 +96,87 @@ or specify a custom confiuration file location:
...
+the authentication of your private zone pollers independent of your on-node agents._
 
-NOTE: if you forget the pass phrase for the ca.pem, just remove `data/ca` and `data/server-certs` and re-run init.
+**NOTE:** the token is traditionally referred to as an **agent** token, but the concept is interchaneable with **monitoring** tokens.
```
Comment (jjbuchan) [inline README.md]: missing a `g` in interchaneable

File: README.md [hunk 3]
```diff
@@ -88,136 +96,87 @@ or specify a custom confiuration file location:
...
+The JSON object payload contains the following attributes:
 
-```
-go get github.com/golang/mock/mockgen
-```
+Name | Description | Validation
+-----|-------------|-----------
+label | A label of your choosing | (Required) Non-empty string
+maximum_checks | Maximum number of checks that can be created in this zone | (Optional) Integer greater than or equal to 1
+maximum_agents | Maximum number of agent/pollers that can join this zone | (Optional) Integer greater than or equal to 1
```
Comment (jjbuchan) [inline README.md]: These two values aren't actually used yet.  We should probably decide whether we are going to add logic to handle it, or just remove the options.

File: README.md [hunk 4]
```diff
@@ -88,136 +96,87 @@ or specify a custom confiuration file location:
...
+disabled | Disable new check creation and agent connection to this zone | (Optional) Boolean, default is false
```
Comment (jjbuchan) [inline README.md]: Can't remember if this is used, so may fall into the same discussion item above.

File: README.md (hunk 3 above)
Comment (itzg) [reply README.md]: I'll remove these since I agree there doesn't seem to be logic that enforces them yet.

File: README.md (hunk 4 above)
Comment (itzg) [reply README.md]: I'll remove it for now.

**Comment by itzg** [comment]
Thanks @jjbuchan , I made all the changes you requested.

**Pull request merged**